README.md

```diff
@@ -42,7 +42,7 @@ Contoso is the Microsoft entity used for "fake demo companies, etc.". The Contos
 Microsoft's open source engineering system for validating the template system, but never used by actual
 product teams.
 
-### dntmit.docs

```

Comment:
I have no idea what this is, but it's the only instance whereas the alternative appears, so I'm inferring it's a typo for it.